lib/Advisor.js

```diff
@@ -111,67 +144,162 @@ function Advisor(options) {
-                change: 0,
-                targets: [selfStatus],
+                // no changes since we have no peers
+                changes: [],
+
+                // Since we are the only instance reporting status
+                // our `value` is the average which means that the
+                // `meanDelta` is `0`
                 meanDelta: 0
             });
-        } else {
-            let maxStatus = null;
-            let minStatus = null;
-            let sum = 0;
-            allStatuses.forEach(function(status) {
-                if (!status) { return; }
-                let value = status.value;
-                if (maxStatus === null || value > maxStatus.value) {
-                    maxStatus = status;
-                }
-
-                if (minStatus === null || value < minStatus.value) {
-                    minStatus = status;
-                }
-
-                sum += value;
-            });
+        }
+
+        // The lowest `value` among _all_ instances
+        let minValue;
+
+        // The highest `value` among _all_ instances
+        let maxValue;
 
-            let mean = sum / allStatuses.length;
-            let delta = maxStatus.value - minStatus.value;
+        // `Status` of _peer_ associated with highest number of connections
+        let maxPeerStatus = null;
 
-            let change = 0;
-            if (delta > _precision && selfStatus.value > mean) {
-                change = 0 - _stepSize;
+        // `Status` of _peer_ associated with lowest number of connections
+        let minPeerStatus = null;
+
+        // The sum of the `value` property across _all_ instances
+        let sum;
+
+        // Number of instances that have `maxCapacity` greater than 0.
+        // We ignore instances that have `maxCapacity` of 0 because these
+        // instances are out-of-service for some reason.
+        let numInstances = 1;
+
+        // Initialize `sum`, `minValue`, and `maxValue` to `value` for
+        // this instance in the cluster
+        sum = minValue = maxValue = selfStatus.value;
+
+        peerStatuses.forEach(function(status) {
+            // ignore instances that are "out-of-service"
+            if (!status || !_isTargetHealthy(status)) {
+                return;
             }
 
-            let advice = new Advice({
-                change: change,
-                targets: [minStatus],
-                meanDelta: selfStatus.value - mean
-            });
+            let value = status.value;
+            if (value > maxValue) {
+                maxValue = value;
+                maxPeerStatus = status;
+            } else if (maxPeerStatus === null) {
+                maxPeerStatus = status;
+            }
 
-            if (change < 0) { advice.reduction = -change; }
+            if (value < minValue) {
+                minPeerStatus = status;
+                minValue = value;
+            } else if (minPeerStatus === null) {
+                minPeerStatus = status;
```

Comment:
Similar to above, it's not the lowest value (because our self value is lower). It's just the lowest of the peers.